app/assets/javascripts/govuk_publishing_components/components/button.js

```diff
@@ -2,4 +2,4 @@
 // = require govuk/components/button/button.js
 window.GOVUK = window.GOVUK || {}
 window.GOVUK.Modules = window.GOVUK.Modules || {}
-window.GOVUK.Modules.GovukButton = window.GOVUKFrontend
+window.GOVUK.Modules.GovukButton = window.GOVUKFrontend.Button
```

Comment:
Hey @andysellick as you took the time to note this on [#2581](https://github.com/alphagov/govuk_publishing_components/pull/2581) and this in dependant - i'm just going to replicate.

--

Thanks for the link that explains this change, but could you add it to the commit message as well to summarise? e.g. `GOVUK-frontend has changed the naming of components JavaScript modules so individual imports are now attached to window.GOVUKFrontend.[ComponentName] instead of window.GOVUKFrontend`